plugins/BEdita/Core/src/Configure/Engine/DatabaseConfig.php

```diff
@@ -57,14 +65,33 @@ public function __construct($applicationId = null)
     }
 
     /**
-     * Read from DB (or cache) $key group of paramenters (see database `config.context`)
+     * Read `$key` parameters group from database using cache.
+     *
+     * @param string!null $key The group of parameters to read from database (see `config.context`).
+     * @return array Parsed configuration values.
+     */
+    public function read($key): array
```

Comment:
I wanted to, but unfortunately we have to implement `ConfigInterface` method  ```public function read($key);``` 
This change would cause a `PHP Fatal error`...